source/extensions/filters/http/transformation/inja_transformer.cc

```diff
@@ -89,47 +102,89 @@ Extractor::Extractor(const envoy::api::v2::filter::http::Extraction &extractor)
   }
 }
 
+absl::optional<std::string>
+Extractor::getDynamicMetadataValue(Http::StreamFilterCallbacks &callbacks) const {
+  Upstream::ClusterInfoConstSharedPtr ci = callbacks.clusterInfo();
+  if (!ci) {
+    ENVOY_STREAM_LOG(debug, "No cluster info found for dynamic metadata", callbacks);
+    return absl::nullopt;
+  }
+
+  const envoy::config::core::v3::Metadata *cluster_metadata{};
+  cluster_metadata = &ci->metadata();
+  const ProtobufWkt::Value &value = Envoy::Config::Metadata::metadataValue(cluster_metadata, SoloHttpFilterNames::get().Transformation, dynamic_metadata_);
```

Comment:
We probably don't want to be restricted to the Transformation filter namespace

Reply:
No longer relevant, as the functionality adding support for dynamic metadata extractor inputs has been removed from this PR